packages/kbn-plugin-helpers/README.md

```diff
@@ -69,5 +69,6 @@ Setting | Description
 ------- | -----------
 `skipArchive` | Don't create the zip file, leave the build path alone
 `buildDestination` | Target path for the build output, absolute or relative to the plugin root
+`buildIgnoreDependencies` | Don't install dependencies defined in package.json into build output
```

Comment:
Maybe `skipInstallDependencies`? (To match `skipArchive`)

Reply:
👍 Felt like it should match `buildDestination`, but i think `skipInstallDependencies` works.